### ISSUE TITLE
tagName on application template

### ISSUE THREAD

**Issue by MatthewPringle** [emberjs/ember.js#14934]
This is a continuation of a previous bug report. Which no longer stands / the suggestions do not work.

I do not want ember inserting body > div.ember-view

I want to use the body as the main view.

How can I achieve this?

**Comment by rwjblue** [comment]
**tldr;** You cannot.

This is not a feature in Ember 2.x (even though it was possible in Ember 1.x). The behavior (using `Ember.View`'s) was deprecated in 1.13 and removed in Ember 2.0.0, however we provided a temporary "legacy views" addon which allowed things to work through Ember 2.4.

It is absolutely possible for someone to submit an RFC to allow applications to opt-out of the wrapping div for the application template.  That RFC needs to work through things like the migration path for folks with CSS that depends on the existing nesting (there are likely many more projects that happen to depend on the current structure than projects that want to avoid it), how we deprecate the existing system, etc.

I'm closing this for now, as it isn't a bug and would be a new feature request.
